frontend/app/src/store/main.ts

```diff
@@ -733,7 +743,10 @@ export class MainStore {
     if (body.price_to_meet) body.price_to_meet = parseInt(body.price_to_meet); // must be an int
 
     try {
-      const [r, error] = await this.doCallToRelay('POST', 'profile', body);
+      let request = "profile";
+      if(this.lnToken) request = "person";
+
+      const [r, error] = await this.doCallToRelay('POST', request, body);
```

Comment:
This will need to call tribes directly, I think an if branch would work fine here

Reply:
The if statement enables it to call the tribe server directly,  the function gets the URL to call from the meInfo.url, the difference between tribe and relay here is just the endpoint name.